modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -29,6 +29,13 @@ ifeval::[{uyuni-content} == true]
 endif::[]
```

Comment:
I still don't see where are the instructions to import the repos from DVD below! Only 6 and 7 are mentioned!

Reply:
See latest commit. I have added the content to the sleses document.